README.md

```diff
@@ -1,52 +1,38 @@
-Copyright (C) <2012> <LexisNexis Risk Data Management Inc.>
+JDBC driver
+=======================
 
-All rights reserved. This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU Affero General Public License as
-published by the Free Software Foundation, either version 3 of the
-License, or (at your option) any later version.
+#### Description
+Java Database Connectivity (JDBC) is a standard Java API that enables Java applications or client tools that support JDBC to access data from a presumably SQL-compliant data source via the SQL language.
 
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-GNU Affero General Public License for more details.
+JDBC makes it possible to write a single database application that can run on different platforms and interact with different database management systems.
 
-You should have received a copy of the GNU Affero General Public License
-along with this program. If not, see <http://www.gnu.org/licenses/>.
+Currently there are JDBC drivers available for interaction with many popular data sources.  This project allows the end user to interact with the HPCC Platform as a data source.  This is achieved by exposing HPCC logical files as RDB tables.
```

Comment:
... HPCC logical files as RDB tables, and HPCC published queries as RDB stored procedures.